core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -906,14 +906,19 @@ public static boolean isValidTableName(CharSequence tableName, int fsFileNameLim
     }
 
     public static int lock(FilesFacade ff, Path path, boolean verbose) {
-        final int fd = ff.openRW(path, CairoConfiguration.O_NONE);
-        if (fd == -1) {
-            if (verbose) {
-                LOG.error().$("cannot open '").utf8(path).$("' to lock [errno=").$(ff.errno()).I$();
+
+        // workaround for https://github.com/docker/for-mac/issues/7004
+        if (Files.VIRTIO_FS_DETECTED) {
+            if (!ff.touch(path)) {
+                LOG.error().$("cannot touch '").utf8(path).$("' to lock [errno=").$(ff.errno()).I$();
```

Comment:
Before this patch, all logging was made under the `verbose` flag check. Should we keep it?

Reply:
good catch, thanks!

Reply:
I cherry-picked the patch from my troubleshooting branch which was always logging (+a bunch more). 
but in prod code it should be guarded by `verbose`.